.github/workflows/ci.yaml

```diff
@@ -2,6 +2,12 @@ name: CI
 
 on: [push, pull_request]
 
+permissions:
+    checks: write
```

Comment:
Can you put this under the specific job, instead of top-level per OpenSSF: https://github.com/ossf/scorecard/blob/main/docs/checks.md#token-permissions

Reply:
Yeah, good catch.